framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1633,7 +1634,7 @@ protected final void addComputedTypeAnnotations(Tree tree, AnnotatedTypeMirror t
      */
     protected void addComputedTypeAnnotations(
             Tree tree, AnnotatedTypeMirror type, boolean iUseFlow) {
-        assert root != null
+        assert root != null || tree instanceof LiteralTree
```

Comment:
This seems wrong to me.  I'm assuming there's a problem because of the LiteralTrees created by the Initialized Fields Checker.  I'll see if I can find a better work around.

Reply:
@smillst That is correct.  The synthetic LiteralTrees lack some information, such as a path in source code.  There are a few hard-coded checks for `LiteralTree` in this pull request.  Fixing that would be nice.

Reply:
I was able to remove these.  See my last commit.